easybuild/scripts/bootstrap_eb.py

```diff
@@ -448,7 +459,9 @@ def main():
     for path in orig_sys_path:
         include_path = True
         # exclude path if it's potentially an EasyBuild package
-        if 'easybuild' in path:
+        if 'easybuild' in path or os.path.exists(os.path.join(path, 'easybuild')):
+            include_path = False
+        if '/vsc_' in path or os.path.exists(os.path.join(path, 'vsc')):
```

Comment:
OK, I'll look into fixing this... Just checking for `vsc/__init__.py` should do it, actually.